lib/metamodel.js

```diff
@@ -19,6 +19,5 @@
\\$|_|\\\\u[0-9A-Fa-f]{4}|\\p{Mn}|\\p{Mc}|\\p{Nd}|\\p{Pc}|\\u200C|\\u200D)*$/u\n  o Boolean isArray default=false\n  o Boolean isOptional default=false\n  o Decorator[] decorators optional\n  o Range location optional\n}\n\nconcept RelationshipProperty extends Property {\n  o TypeIdentifier type\n}\n\nconcept ObjectProperty extends Property {\n  o String defaultValue optional\n  o TypeIdentifier type\n}\n\nconcept BooleanProperty extends Property {\n  o Boolean defaultValue optional\n}\n\nconcept DateTimeProperty extends Property {\n}\n\nconcept StringProperty extends Property {\n  o String defaultValue optional\n  o StringRegexValidator validator optional\n  o StringLengthValidator lengthValidator optional\n}\n\nconcept StringRegexValidator {\n  o String pattern\n  o String flags\n}\n\nconcept StringLengthValidator {\n  o Integer minLength optional\n  o Integer maxLength optional\n}\n\nconcept DoubleProperty extends Property {\n  o Double defaultValue optional\n  o DoubleDomainValidator validator optional\n}\n\nconcept DoubleDomainValidator {\n  o Double lower optional\n  o Double upper optional\n}\n\nconcept IntegerProperty extends Property {\n  o Integer defaultValue optional\n  o IntegerDomainValidator validator optional\n}\n\nconcept IntegerDomainValidator {\n  o Integer lower optional\n  o Integer upper optional\n}\n\nconcept LongProperty extends Property {\n  o Long defaultValue optional\n  o LongDomainValidator validator optional\n}\n\nconcept LongDomainValidator {\n  o Long lower optional\n  o Long upper optional\n}\n\nconcept AliasedType{\n  o String name\n  o String aliasedName\n}\nabstract concept Import {\n  o String namespace\n  o String uri optional\n}\n\nconcept ImportAll extends Import {\n}\n\nconcept ImportType extends Import {\n  o String name\n}\n\nconcept ImportTypes extends Import {\n  o String[] types\n  o AliasedType[] aliasedTypes optional\n}\n\nconcept Model {\n  o String namespace\n  o String sourceUri optional\n  o String concertoVersion optional\n  o Import[] imports optional\n  o Declaration[] declarations optional\n  o Decorator[] decorators optional\n}\n\nconcept Models {\n  o Model[] models\n}\n\nabstract concept ScalarDeclaration extends Declaration {\n}\n\nconcept BooleanScalar extends ScalarDeclaration {\n  o Boolean defaultValue optional\n}\n\nconcept IntegerScalar extends ScalarDeclaration {\n  o Integer defaultValue optional\n  o IntegerDomainValidator validator optional\n}\n\nconcept LongScalar extends ScalarDeclaration {\n  o Long defaultValue optional\n  o LongDomainValidator validator optional\n}\n\nconcept DoubleScalar extends ScalarDeclaration {\n  o Double defaultValue optional\n  o DoubleDomainValidator validator optional\n}\n\nconcept StringScalar extends ScalarDeclaration {\n  o String defaultValue optional\n  o StringRegexValidator validator optional\n  o StringLengthValidator lengthValidator optional\n}\n\nconcept DateTimeScalar extends ScalarDeclaration {\n  o String defaultValue optional\n}\n";
```

Comment:
This looks shortened, I would have expected to become longer.

Reply:
I think its the regex. In the github diff viewer, older code is taking more lines than the new code.

On my vscode diff viewer things seemed fine.